src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -251,6 +268,16 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
   const navBarScheduleClassName = "NavBar__schedule-dropdown";
 
+  const scheduleArrowClasses = classNames({
+    "arrow-up": isEventScheduleVisible,
+    "arrow-down": !isEventScheduleVisible,
+  });
+
+  const dropdownArrowClasses = classNames({
+    "arrow-up": isDropdownShown,
+    "arrow-down": !isDropdownShown,
+  });
+
```

Comment:
Why are classes not following BEM?
Any reason they should be added outside `.NavBar {}` block?

A solution in line with common project style would be:
```suggestion
  const scheduleArrowClasses = classNames({
    "NabBar__schedule-arrow--up": isEventScheduleVisible,
    "NabBar__schedule-arrow--down": !isEventScheduleVisible,
  });

  const dropdownArrowClasses = classNames({
    "NabBar__dropdown-arrow--up": isDropdownShown,
    "NabBar__dropdown-arrow--down": !isDropdownShown,
  });

```

Reply:
fixed in that PR https://github.com/sparkletown/sparkle/pull/2387